test/python/test_softplus.py

```diff
@@ -0,0 +1,55 @@
+# ==============================================================================
+#  Copyright 2018-2019 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+"""nGraph TensorFlow softplus test
+
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pytest
+
+import numpy as np
+import tensorflow as tf
+
+from common import NgraphTest
+
+
+class TestSoftplus(NgraphTest):
+
+    def test_softplus(self):
+        x = tf.placeholder(tf.float32, shape=(2, 3))
+        y = tf.placeholder(tf.float32, shape=(2, 3))
+        z = tf.placeholder(tf.float32, shape=(2, 3))
+
+        a = x + y + z
+        b = x + y + z
+        c = a * b
+        d = tf.nn.softplus(c)
```

Comment:
Lets run this pytest locally with `NGRAPH_TF_LOG_PLACEMENT=1` to ensure that a cpp softplus is produced (instead of breaking down into other ops like exp/log etc).